app/models/job.rb

```diff
@@ -1,4 +1,19 @@
 class Job < ActiveRecord::Base
   validates :title, :level_of_interest, presence: true
   belongs_to :company
+  belongs_to :category
+  has_many :comments
+
+  def show_by_count_of_jobs_by_level_of_interest
```

Comment:
same here, this should probably be a class method rather than an instance method 👍